core/arch/x86_64/kernel/thread_private.h

```diff
@@ -169,7 +169,7 @@ void thread_unlock_global(void);
  * The purpose of this function is to request services from non-secure
  * world.
  */
-#define THREAD_RPC_NUM_ARGS     4
+#define THREAD_RPC_NUM_ARGS     6
```

Comment:
Why do we still need 6 args number here?

Reply:
ARM doesn't use parameter a4, a5, just cut them. 

	/*
	 * Return from RPC to request service of a foreign interrupt must not
	 * get parameters from non-secure world.
	 */
	if (threads[n].flags & THREAD_FLAGS_COPY_ARGS_ON_RETURN) {
		copy_a0_to_a3(&threads[n].regs, a0, a1, a2, a3);
		threads[n].flags &= ~THREAD_FLAGS_COPY_ARGS_ON_RETURN;
	}